ens.asciidoc

```diff
@@ -1,7 +1,12 @@
 == Ethereum Name Service (ENS)
 
-The _Ethereum Name Service (ENS)_ is one of the first Ethereum DApps, offering a decentralized name registry for Ethereum. It can be compared to the Domain Name System (DNS), but unlike DNS which is partially centralized, ENS is completely decentralized.
+You can design the best smart contract in the world, but if you don't provide a good interface for users, they won't be able to access it. Contracts on the blockchain can be considered the backend of your application, and you can write the front-end in any language or platform you feel comfortable. Most of the apps you'll read about on this book currently use a traditional server model to serve that front-end, but if you want to truly decentralize your application, you can also use decentralized hosting services like bit torrent, swarm or IPFS to also host the files required for the front-end.
 
+In a traditional server model of the web, you access content by describing the address where you can find the content. For instance 91.198.174.192 is the IP address of Wikipedia. That number has no information on what sort of content you will find, only the address of the computers that will serve you wikipedia's content. Content-addressing systems are different that their address reflect the content itself not its location. For instance, the url `QmfXeApFP1mCrMTT1JX5XLtUvLxS5eNsegqDpBYUZPhXEQ` actually describes an image that you can download from any computer on the IPFS network. Different standards, like IPFS or Swarm will have different hashes, but they all describe the content not the address itself. This allows more flexibility and decentralization because as long as a single node in your network has a copy of that image (or is connected to a node that has a copy of it), they can access it.
+
+This process allows acessing any files that will make up your interface, html, javascript, css, images, etc in a decentralized manner, but has a huge drawback: urls are not very human friendly. They consist of large mumbles of letters and numbers, which are neither memorable or easy to type. On the traditional IP-based internet, this was solved by Domain Naming system (DNS), an hierarchical system ultimately centralized on ICANN, a US-based organization that controls the distribution of top level domaisn, like .com or .org. On the blockchain an attempt to solve the same problem but in a decentralized manner is the Ethereum Naming System (ENS).
+
+Another advantadge of a naming system is that it can also be used as a aliases for receiving funds or identifiers. As an example, the Ethereum Foundation donation contract address is `0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359` or, in a wallet that supports ENS, it's simply `ethereum.eth`.
```

Comment:
My English is not native but ...

"it can also be used as a aliases"
-->
"it can also be used as an alias"